tool/teleport/common/teleport.go

```diff
@@ -71,6 +71,8 @@ func Run(cmdlineArgs []string, testRun bool) (executedCommand string, conf *serv
 	start.Flag("debug", "Enable verbose logging to stderr").
 		Short('d').
 		BoolVar(&ccf.Debug)
+	start.Flag("disable-tls", "Disable TLS use").
```

Comment:
this should go with `insecure` prefix, `insecure-disable-tls`

Reply:
@klizhentas hehe, I suggested the same thing, except a shorter version: `--insecure-no-tls`

Reply:
sounds good, let's pick version from  @kontsevoy